script/watch.js

```diff
@@ -78,9 +85,12 @@ function runMochaTests(tests) {
       tests,
       showErrors
     });
-    forked.on('message', (requiredProjectFiles) => {
+    forked.on('message', (message) => {
+      if (message.error) {
+        reject(message.error);
+      }
       // sent the watcher the files to watch
-      fulfill(requiredProjectFiles);
+      fulfill(message);
```

Comment:
The comment seems confusing now, since this isn't the files to watch? Or is it still, if there's not an error?

Reply:
This is pretty opaque. I updated the variable named to `requiredFiles` and used destructuring assignment on the message object so that's clearer what's in it. 

I don't know if this is the idiomatic way to communicate errors from a forked process. The `error` seems to be for setup problems and not something I can fire from code.